### PR TITLE
Add my card information.

### PR DESCRIPTION
**Pull request by amersfoort3232**
Add my card to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,7 +114,30 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:117]:
```suggestion

        <!-- ________ *Amersfoort* Contributor card START ________  -->
```

File: index.html [hunk 2]
```diff
@@ -114,7 +114,30 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+                <a href="https://www.w3schools.com/" target="_blank" title="Link to w3schools">w3schools</a>
+              </li>
+              <li>
+                <a href="https://www.freecodecamp.org/" target="_blank" title="Link to FreeCodeCamp">FreeCodeCamp</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:140]:
```suggestion
        <!-- ________*Amersfoort* Contributor card END ________  -->
        
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
kindly do the recommended changes and commit.

**Review by nzaih1999 — APPROVED** [review]

**Comment by nzaih1999** [comment]
Hello @amersfoort3232 I did some little changes on your PR. Your changes are now ready for merge. Thanks for your contribution. Happy Coding

**Pull request merged**